cloudevents/extensions/loglevel.md

```diff
@@ -0,0 +1,163 @@
+# Log Level Extension
+
+## Abstract
+This extension defines attributes that MAY be included within a CloudEvent
+to describe the "level" or "importance" of an event in relation to other events
+produced by a unique event source.
+
+Often systems produce events in form of logs, and these types of events usually share
+a common concept of "log-level". This extension aims to provide a standard way for 
+describing this property in a language agnostic form. 
+
+Sharing a common way to describe importance of events allows for better monitoring 
+systems, tooling and general log consumption.
+
+In addition to defining the extension attributes, this document defines 
+the concept of [Log Level Bindings](#log-level-bindings) - a standard way to map 
+well-known log level names from different languages and frameworks 
+onto the `loglevelname` and `loglevelnum` values defined in this spec. 
+
+## Attributes
+
+### loglevelname 
+
+- Type: `String`
+- Description: Human readable representation of the log level
+- Constraints
+  - OPTIONAL
+  - if present, MUST be a non-empty string
+  - SHOULD be lowercase
+  - RECOMMENDED values include `emergency`, `alert`, `critical`, `error`, 
+    `warning`, `notice`, `info`, `debug`, and `verbose`, but  others MAY be used.
+  - SHOULD have a one-to-one relationship with a distinct `loglevelnum` in the scope of
+   the `source`
+
+### loglevelnum 
+
+- Type: `Integer`
+- Description: A numerical representation of the level of importance of the event. 
+  Conforms to the severity ordering specified by 
+  [RFC5424](https://www.rfc-editor.org/rfc/rfc5424.html#section-6.2.1): severity of
+  all levels is assumed to be numerically ascending from most important to least
+  important.
+  This specification does not define "importance" or meaning of each value other than
+  an event with a lower numerical value MUST be of more importance than an event with
+  a higher numerical value in the same `source`.
+
+- Constraints
+  - OPTIONAL
+  - if present, MUST NOT be negative
+  - RECOMMENDED values include
+    - `0` for `emergency`
+    - `1` for `alert`
+    - `2` for `critical`
+    - `3` for `error`
+    - `4` for `warning`
+    - `5` for `notice`
+    - `6` for `info`
+    - `7` for `debug`
+    - `8` for `verbose`
+
+## Log Level Bindings
+
+A standard way to map well-known log level names from different languages and 
+frameworks onto the `loglevelname` and `loglevelnum` values defined in this spec. 
+
+Each binding defines the unique log name and/or log number in a framework and the 
+corresponding RECOMMENDED `loglevelname` and `loglevelnum`.
+
+### RECOMMENDED Log Bindings
+
+#### Syslog
+
+All syslog log records SHOULD assign the severity level to `loglevelnum`. 
+In addition to that The corresponding `loglevelname`s to each of the severity values
```

Comment:
s/The/the/
Is the end of this sentence missing?

Reply:
what do you mean when you write `s/The/the/` ?

Reply:
Oh, you mean substitute The with the
like in vim

Reply:
:-) yes like vim